### ISSUE TITLE
Label formatting: add ability to add references to label block lines

### ISSUE THREAD

**Issue by arbolitoloco** [Symbiota/community-board#614]
Feature suggested by @mickley:

> it would be nice if there was a different class name included for each label line (e.g., line1), rather than just fieldblock, so that each line could be styled individually with some ease.

I'm thinking this would be in the form of ids and not classes. Would need to be worked in the format editor, as well as the visual interface editor.

**Comment by mickley** [comment]
Yes, ids would work fine. It would be great to have something like that in the visual editor, but even just adding the ID to the divs would allow a lot with an added CSS file. Thank you @arbolitoloco!
